src/OrchardCore.Modules/OrchardCore.OpenId/Settings/OpenIdServerSettings.cs

```diff
@@ -9,9 +9,12 @@ public class OpenIdServerSettings
     {
         public TokenFormat AccessTokenFormat { get; set; }
         public Uri Authority { get; set; }
-        public StoreLocation? CertificateStoreLocation { get; set; }
```

Comment:
This is a breaking change that will break existing OC deployments but I'm convinced we should adopt it, as it will avoid ambiguities in the future.